pychromecast/__init__.py

```diff
@@ -349,20 +349,20 @@ def reboot(self):
         """ Reboots the Chromecast. """
         reboot(self.host)
 
-    def volume_up(self):
-        """ Increment volume by 0.1 unless it is already maxed.
+    def volume_up(self, delta=0.1):
+        """ Increment volume by 0.1 (or delta) unless it is already maxed.
         Returns the new volume.
 
         """
-        volume = round(self.status.volume_level, 1)
-        return self.set_volume(volume + 0.1)
+        volume = round(self.status.volume_level, 2)
```

Comment:
You can't just  adjust the rounding to now match the delta that you're planning to use.